c2corg_ui/static/js/search.js

```diff
@@ -98,6 +85,12 @@ app.SearchController = function($rootScope, $compile, $attrs, apiUrl, gettextCat
    */
   this.selectHandler;
 
+/**
```

Comment:
The indentation of the comment block is wrong, should be shifted 2 spaces to the right.